lib/phonebook/providers/association_member_list_provider.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:myecl/auth/providers/openid_provider.dart';
+import 'package:myecl/phonebook/class/complete_member.dart';
+import 'package:myecl/phonebook/providers/association_provider.dart';
+import 'package:myecl/phonebook/repositories/association_member_repository.dart';
+import 'package:myecl/tools/providers/list_notifier.dart';
+import 'package:myecl/tools/token_expire_wrapper.dart';
+
+class AssociationMemberListNotifier extends ListNotifier<CompleteMember> {
+  final AssociationMemberRepository associationMemberRepository =
+      AssociationMemberRepository();
+  AssociationMemberListNotifier({required String token})
+      : super(const AsyncValue.loading()) {
+    associationMemberRepository.setToken(token);
+  }
+
+  Future<AsyncValue<List<CompleteMember>>> loadMembers(String associationId, String year) async {
+    dynamic members = await loadList(() async => associationMemberRepository.getAssociationMemberList(associationId, year));
```

Comment:
Make it final instead of dynamic, or even return it directly

Reply:
Fixed in next commit